include/cutlass/epilogue/thread/linear_combination_gelu.h

```diff
@@ -133,7 +133,7 @@ class LinearCombinationGELU {
 
   /// Functionally required for serial reduction in the epilogue
   CUTLASS_HOST_DEVICE
-  void set_k_partition(int k_partition) {
+  void set_k_partition(int k_partition, int k_partition_count) {
```

Comment:
The signatures for set_k_partition for all other activations are (int, int), and the invocation from [gemm.h](https://github.com/NVIDIA/cutlass/blob/master/include/cutlass/gemm/kernel/gemm.h#L289), [gemm_universal.h](https://github.com/NVIDIA/cutlass/blob/master/include/cutlass/gemm/kernel/gemm_universal.h#L483), [sparse_gemm.h](https://github.com/NVIDIA/cutlass/blob/master/include/cutlass/gemm/kernel/sparse_gemm.h#L322) all assumes (int, int).
If we don't add, build with LinearCombinationGELU would fail with:
```
cutlass/include/cutlass/gemm/kernel/gemm.h(289): error: too many arguments in function call
```

Reply:
Yeah, I encountered this build error as well.

Reply:
gotcha.

Would you please add `CUTLASS_UNUSED(k_partition_count)` to mute a warning? https://github.com/NVIDIA/cutlass/blob/master/include/cutlass/cutlass.h#L34

Thank you.  I will run some internal test before merging this.  We have some backlog these days.  So it may take a while.

Reply:
Update. Thanks for the comments.